src/app/common-ui/Form/components/LabeledInput.jsx

```diff
@@ -39,6 +36,15 @@ const Input = styled.input`
             border-bottom-color: #1c83c6;
           }
         `}
+  ${({ isDisabled }) =>
+    isDisabled
+      ? css`
+          color: rgba(0, 0, 0, 0.5);
```

Comment:
Declare `--black50` in `src/app/style.scss` and reuse from that
`color: var(--black50)`